test/support/model.ex

```diff
@@ -8,17 +8,15 @@ defmodule ValidField.Support.Model do
     field :title, :string
     field :password, :string
     field :password_confirmation
-    field :date_of_birth, Ecto.DateTime
+    field :date_of_birth, Ecto.Date
 
     timestamps
   end
 
-  @required_fields ~w(first_name)
-  @optional_fields ~w(last_name title password password_confirmation)
-
   def changeset(model, params \\ :empty) do
     model
-    |> cast(params, @required_fields, @optional_fields)
+    |> cast(params, [:first_name, :last_name, :title, :password, :password_confirmation, :date_of_birth])
```

Comment:
`cast/4` is being deprecated in favor of `cast/3` with `validate_required`.  Our required fields should be listed in `validate_required`